src/index.jsx

```diff
@@ -35,17 +35,18 @@ import getReduxStore from './reduxStore';
 import { ReduxNavBar, ReduxTopBar, ReduxFooter } from './Layout/reduxer';
 import ReduxQueryNode, { submitSearchForm } from './QueryNode/ReduxQueryNode';
 import { basename, dev, gaDebug, workspaceUrl, workspaceErrorUrl,
-  indexPublic, useGuppyForExplorer, explorerPublic,
+  indexPublic, useGuppyForExplorer, explorerPublic, resourceBrowserPublic,
 } from './localconf';
 import Analysis from './Analysis/Analysis';
 import ReduxAnalysisApp from './Analysis/ReduxAnalysisApp';
-import { gaTracking, components } from './params';
+import { gaTracking, components, config } from './params';
```

Comment:
i would prefer to have some parameter like `enableResourceBrowser` in `localconf` and read that in, rather than importing the whole `config` into here